app/models/heads_of_term.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class HeadsOfTerm < ApplicationRecord
```

Comment:
This is kind of a minor thing, but I wonder if it might end up being confusing that this is called `HeadsOfTerm` rather than `HeadsOfTerms`. But it might be more trouble than it's worth to try to make it work properly with Rails' pluralisations. I think we can add a custom inflection but I don't know how well it works if we want to make the plural and the singular the same. (And having a `HeadsOfTermsesController` would be even worse.)

Reply:
Yeah this just seems like a big old can of worms 😬